src/Umbraco.Web.UI.Client/src/packages/media/media/components/input-image-cropper/input-image-cropper.element.ts

```diff
@@ -50,8 +50,13 @@ export class UmbInputImageCropperElement extends UmbFormControlMixin<
 	@state()
 	fileUnique?: string;
 
+	@state()
+	private _accept?: string;
+
 	#manager?: UmbTemporaryFileManager;
 
+	#temporaryFileConfig = new UmbTemporaryFileConfigRepository(this);
+
 	constructor() {
 		super();
 		this.#manager = new UmbTemporaryFileManager(this);
```

Comment:
Noting here, that the `UmbTemporaryFileManager` class creates its own instance of `UmbTemporaryFileConfigRepository`, so we're effectively making the same request for the configuration twice.

Could we extend `UmbTemporaryFileManager` to retrieve the configuration values? 🤔

Reply:
Ohhh, I just moved the initialization of that away from the constructor into the property itself. That seems to have solved the issue actually! But yes, definitely something to look into there...